Makefile

```diff
@@ -148,6 +148,7 @@ prod: globals check-env-vars ## Set Environment to Production
 	$(eval export ENABLE_PAAS_DASHBOARD=true)
 	$(eval export DEPLOY_RUBBERNECKER=true)
 	$(eval export DEPLOY_ENV=prod)
+	$(eval export TEST_HEAVY_LOAD=false)
```

Comment:
I think we should invert the sense of this variable - ie have it default to false, and then set it to true for dev/ci/staging. That will make it consistent with other variables, and make it less likely that this gets deployed to prod by mistake.